### ISSUE TITLE
Pigeon brothers wheels are invisible

### ISSUE THREAD

**Issue by AliceDTRH** [scemino/engge#273]
During the start of the game when the pigeon "brothers" van drives, the wheels are invisible.

**Comment by scemino** [comment]
Thank you for message, you're right.
I'm working on engge 2 which is a total rewrite of engge, and it will fix this issue as you can see here:
<img width="1000" alt="Capture d’écran 2022-08-20 à 20 57 00" src="https://user-images.githubusercontent.com/643384/185762385-fa0d0375-1582-47da-8f5d-07a0195a7a27.png">

And a video:
https://youtu.be/5tHoBUICSJk

**Comment by AliceDTRH** [comment]
Yay, wheels! Looking forward to it.

**Comment by eblanca** [comment]
> I'm working on engge 2 which is a total rewrite of engge, and [cut]

Is this new implementation hosted on github as well?

**Comment by scemino** [comment]
> > I'm working on engge 2 which is a total rewrite of engge, and [cut]
> 
> Is this new implementation hosted on github as well?

Yes, it will be available when the project will be more mature, and I don't know yet how to call this project `engge`, `engge 2`, something else ?

**Comment by eblanca** [comment]
>  and I don't know yet how to call this project `engge`, `engge 2`, something else ?

Well, as engge 1 had two 'g's, then maybe version 2 should have three, as in enggge :-)
Or maybe the trivial thimblefreed park will make people relate the engine with the game instantly!

**Comment by scemino** [comment]
> Or maybe the trivial thimblefreed park will make people relate the engine with the game instantly!

👍 I like it or why not `Freembleweed Park` ?

**Comment by AliceDTRH** [comment]
Won't that annoy the original devs? Like, trademark wise and such.

**Comment by neuromancer** [comment]
> Won't that annoy the original devs?

It could be the case so just to be sure it is suggested to use something "different". You could also try TPVM (as opposed as ScummVM) or something like that.
